engine/src/io/data_parser/ArgsUtil.cpp

```diff
@@ -242,11 +249,53 @@ std::string getDataTypeName(DataType dataType) {
 	case DataType::JSON: return "json"; break;
 	case DataType::CUDF: return "cudf"; break;
 	case DataType::DASK_CUDF: return "dask_cudf"; break;
+	case DataType::MYSQL: return "mysql"; break;
+	case DataType::POSTGRESQL: return "postgresql"; break;
+	case DataType::SQLITE: return "sqlite"; break;
 	default: break;
 	}
 
 	return "undefined";
 }
 
+sql_info getSqlInfo(std::map<std::string, std::string> &args_map) {
+  // TODO percy william maybe we can move this constant as a bc.BlazingContext config opt
+  const size_t DETAULT_TABLE_BATCH_SIZE = 1000;
```

Comment:
That feels like a very small number depending on the latency and overhead of fetching lets say 10k or 100k rows vs fetching 1k 10 or 100 times

Reply:
now is 100k, done